.env

```diff
@@ -0,0 +1,12 @@
+# MySQL Configuration
+MYSQLDB_ROOT_PASSWORD=root1234
+MYSQLDB_DATABASE=bookstore
+MYSQLDB_USER=bookuser
+MYSQLDB_PASSWORD=bookpass
+MYSQLDB_LOCAL_PORT=3307
+MYSQLDB_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8080
+SPRING_DOCKER_PORT=8080
+DEBUG_PORT=5005
```

Comment:
git rm --cached .env - .env should never be in version control, add .env.template instead with actual values removed